### DISCUSSION TITLE
In-Battle Gimmick Models

### DISCUSSION THREAD

**Discussion by WhatAUsernameIHave** [Deijin27/RanseiLink#55] [Q&A]
Is it currently possible to edit gimmick objects in-battle models? Like changing Pokemari Balls to a blank round sphere as an example. I’d presume they have the same sprite sheet as Pokémon Battle Models.

**Comment by Deijin27** [comment]
They are 2d sprites mapped to 3d models like the pokemon. But some gimmicks are fully 3d models, so I cant just extract the textures like with pokemon for all of them (well, it's possible but it would lock those slots into being 2d and require bespoke specifications for each one which is no fun).

What I do with the map exporter would probably work. I'll investigate the feasability of it.
